<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transDialog</class>
 <widget class="QDialog" name="transDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(23, 6, 77);
border-radius: 20px;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>191</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(23, 6, 77);
border-radius:15px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>how much?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>who?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_who">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_how">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:rgb(23, 6, 77);

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton_done">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
background-color:rgb(23, 6, 77);

</string>
    </property>
    <property name="text">
     <string>done</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>group name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_groupname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
